<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRmoPropPage</class>
 <widget class="QWidget" name="QRmoPropPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rmo</string>
  </property>
  <widget class="QGroupBox" name="groupBoxServer">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>44</y>
     <width>177</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Connect to server</string>
   </property>
   <widget class="QLineEdit" name="lineEditAddress">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>24</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddress">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>26</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>52</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPort">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>16000</number>
    </property>
    <property name="value">
     <number>7776</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelRmoId">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>16</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rmo ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRmoId">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>12</y>
     <width>45</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>OnButtonColorClicked()</slot>
  <slot>OnButtonFontClicked()</slot>
 </slots>
</ui>
